components-pom/pom.xml

```diff
@@ -309,6 +309,11 @@
       <artifactId>jersey-common</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
```

Comment:
do we realy need a jersey-client dependency in all components using celements-component pom as parent pom?
I think we should only "activate" the dependency in the pom file of the component needing the jersey-client dependency.